query.go

```diff
@@ -545,14 +546,26 @@ func queryDriverProperties(
 	m := make(map[driver.Driver][]prop.Media)
 
 	for _, d := range drivers {
-		if d.Status() == driver.StateClosed {
+		var status string
+		isAvailable, err := driver.IsAvailable(d)
+		if errors.Is(err, availability.ErrUnimplemented) {
+			s := d.Status()
+			status = string(s)
+			isAvailable = s == driver.StateClosed
+		} else if err != nil {
+			status = err.Error()
+		}
```

Comment:
This should be simpler once we add support to get the availability status for other drivers.

Reply:
yeah for sure